### PR TITLE
[SMALLFIX] Display persistence state on the UI

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

**Comment by yupeng9** [comment]
Jenkins, test this please

File: servers/src/main/java/tachyon/web/UiFileInfo.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public UiFileInfo(LocalFileInfo fileInfo) {
     mUserName = "";
     mGroupName = "";
     mPermission = FileSystemPermission.getNoneFsPermission().toShort();
-    mIsPersisted = false;
+    mPersistenceState = "NOT_PERSISTED";
```
Comment (apc999) [inline servers/src/main/java/tachyon/web/UiFileInfo.java]: why not `PersistenceState.NOT_PERSISTED.name()` ?

Comment (yupeng9) [inline servers/src/main/java/tachyon/web/UiFileInfo.java]: done

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**